src/Tooling/Bit.Tooling.Templates/TodoTemplate/Shared/Resources/ErrorStrings.resx

```diff
@@ -252,4 +252,7 @@
   <data name="WaitForResetPasswordEmailResendDelay" xml:space="preserve">
     <value>You have already requested for the reset password email for '{0}'. Try again later.</value>
   </data>
+  <data name="EmailAlreadyConfirmed" xml:space="preserve">
+    <value>Your email already confirmed.</value>
```

Comment:
The value of this message is better to be like this:
`Your email is already confirmed.`